jenkins-setup/init.groovy

```diff
@@ -0,0 +1,5 @@
+import hudson.model.DownloadService;
+println 'Downloading the plugins meta info...'
+for (d in DownloadService.Downloadable.all()) {
+    d.updateNow();
+}
```

Comment:
Missed EOF blank line here

Reply:
I would highly recommend using https://editorconfig.org/. You copy [this file](https://github.com/jenkinsci/folder-auth-plugin/blob/master/.editorconfig) directly and configure your IDE to ensure new lines are added automatically on every save.